packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +39,22 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        if (externalPkg && externalPkg.name) {
+            try {
+                const packages = ['init', 'info', 'migrate', 'serve'];
```

Comment:
No need hardcore that, we already have them in https://github.com/webpack/webpack-cli/blob/next/packages/webpack-cli/lib/utils/cli-flags.js#L22